### PR TITLE
Login

### PR DESCRIPTION
**Pull request by poojakelkar**
chrome-capture-2022-3-5 (1)

### CODE REVIEW

**Review by jeetbhalani15 — APPROVED** [review]
code looks clean  with proper naming with good UI

File: src/pages/LoginPage/index.jsx [hunk 1]
```diff
@@ -1,7 +1,82 @@
...
+import { toast } from "react-toastify";
 
 const Login = () => {
-  return <div>login page</div>;
+  const navigate = useNavigate();
+  const [formData, setFormData] = useState({
+    username: "",
+    password: "",
+  });
+  console.log({ formData });
```
Comment (akash-1318) [inline src/pages/LoginPage/index.jsx:23]: Please remove console logs.

**Review by akash-1318 — COMMENTED** [review]
Just added a simple point to improve else the code looks good with proper naming conventions and formatting.

File: src/pages/LoginPage/index.jsx (hunk 1 above)
Comment (poojakelkar) [reply src/pages/LoginPage/index.jsx:23]: sure thanks

**Pull request merged**